src/lib.rs

```diff
@@ -52,10 +53,11 @@ use zkevm_opcode_defs::{BinopOpcode, RetOpcode};
 
 /// Run a vm program from the given path using a custom state.
 /// Returns the value stored at storage with key 0 and the final vm state.
-pub fn program_from_file(bin_path: &str) -> Vec<U256> {
-    let program = std::fs::read(bin_path).unwrap();
-    let encoded = String::from_utf8(program.to_vec()).unwrap();
-    let bin = hex::decode(&encoded[2..]).unwrap();
+pub fn program_from_file(bin_path: &str) -> Result<Vec<U256>, EraVmError> {
+    let program = std::fs::read(bin_path)?;
+    let encoded =
+        String::from_utf8(program.to_vec()).map_err(|_| EraVmError::IncorrectBytecodeFormat)?;
+    let bin = hex::decode(&encoded[2..]).map_err(|_| EraVmError::IncorrectBytecodeFormat)?;
```

Comment:
We should probably check that `encoded[..2] == "0x"` as well.

Reply:
Done!